spec/lib/deploy_complexity/git_spec.rb

```diff
@@ -7,8 +7,11 @@
   describe ".safe_name" do
     subject { DeployComplexity::Git.method(:safe_name) }
     its(['origin/master']) { is_expected.to eq 'master' }
+    its(["origin/master\n"]) { is_expected.to eq 'master' }
+    its(["origin/master\n\n"]) { is_expected.to eq "master\n" }
```

Comment:
This can be surprising, though I'm not sure if the use of `#chomp` over `#strip` was intentional.

Reply:
it wasn't! It should probably be `#strip`.